developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -26,17 +27,18 @@ To install the TypeScript client v3, follow these steps:
 
    - The minimum version of Node supported by the v3 client is Node 18. 
 
-1. Install the new client package.
+2. Install the new client package.
```

Comment:
@malgamves You don't need to change the numbers here. They were rendering correctly [staging](https://tsv3-content-changes--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/client-libraries/typescript/v2_v3_migration#install)

The 1. style means you don't need to renumber later if anything changes

Reply:
ahh TIL, i'll change that thanks!